CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## [Unreleased]
 
+- Remove unused Rubinius and Ruby 1.9.2 codepaths (https://github.com/heroku/heroku-buildpack-ruby/pull/1440)
```

Comment:
I don't think you have to document this, since there doesn't seem to be a way for customers to be impacted by the change. But, I also don't think documenting it hurts anything. I'm happy either way now that I have the context.